hooks/queries/sns.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from '@tanstack/react-query';
+import { snsAPI } from 'api/sns';
+import { AxiosError, AxiosResponse } from 'axios';
+import { FollowListType } from 'typings/sns';
+
+export const useGetFollowerList = (accountId: string) => {
+  const { data, isLoading } = useQuery<
+    AxiosResponse<FollowListType>,
+    AxiosError
+  >(['useGetFollowerList'], () => snsAPI.getFollowerList(accountId));
+  return { isLoading, data };
+};
```

Comment:
```suggestion
return useQuery<
    AxiosResponse<FollowListType>,
    AxiosError<ResponseError>
  >(['useGetFollowerList'], () => snsAPI.getFollowerList(accountId));
};
```

전 이렇게 바로 useQuery의 return값을 훅의 return 값으로 변경하는게 더 좋을 것 같아요!
굳이 data랑 isLoading만을 return할 필요는 없다고 보입니다.

그리고 제가 `ResponseError`라는 타입을 생성해주었는데 사용하시면 될 것 같아요!

Reply:
좋습니다! suggestion 그대로 커밋할게요~!
확실히 useQuery return 값을 제어할 필요 없이 넘겨주는게 사용 시 더 편할 것 같네요👍